core/components/atoms/button/button.js

```diff
@@ -224,9 +227,11 @@ Button.Element = styled.button`
   pointer-events: ${props => (props.disabled || props.loading || props.success ? 'none' : null)};
   transition: border-color ${misc.animationDuration}, background ${misc.animationDuration};
 
-  ${Icon.Element}, ${StyledSpinner} {
-    position: relative;
-    top: -1px;
+  display: flex;
```

Comment:
@francocorreasosa This changes the height of the buttons as well, we might have to tweak those

(chromatic shows the diff)

Reply:
Yep, looking into that right now